pkg/policy/source.go

```diff
@@ -37,6 +38,16 @@ func DetectPolicy(name string, subPolicy string) (*Policy, bool, error) {
 	}, true, nil
 }
 
+func classifyError(source string, err error) error {
+	matched, _ := regexp.MatchString("subdir .+? not found", err.Error())
+	formattedError := fmt.Errorf("failed to  %s: %w", source, err)
```

Comment:
double space between `to` and `%s`

Reply:
Thanks! The error should not change actually. Fixed in https://github.com/cloudquery/cloudquery/pull/701/commits/60d5f565e823f3fb8a1a22e8f337f215680ed2f1